_config.yml

```diff
@@ -120,8 +120,8 @@ repos:
 google_analytics_ua: UA-48605964-19
 
 back_link:
-  url: "https://pages.18f.gov/guides/"
-  text: Read more 18F Guides
+  url: https://guides.18f.gov/guides
```

Comment:
@wslack should this just be `https://guides.18f.gov` given what we were talking about earlier in `wg-documentation`?